R/compress.R

```diff
@@ -1,4 +1,8 @@
 #' Return a compressed image
```

Comment:
You may want to add your name as author here. Otherwise the function looks good and well documented! good job!